numpy/core/src/common/simd/sse/sse.h

```diff
@@ -46,6 +46,8 @@ typedef npyv_m128ix3 npyv_s32x3;
 typedef npyv_m128ix3 npyv_u64x3;
 typedef npyv_m128ix3 npyv_s64x3;
 
+typedef npyv_m128ix3 npyv_divisor_t;
```

Comment:
We don't follow STD style here `_t` also you will have to set a type divisor for each data type to make it compatible with vsx and neon. why not just using multi-vector types directly as it was before?